sequential/seq2pat.py

```diff
@@ -405,10 +456,10 @@ def get_patterns(self, min_frequency: Num) -> List[list]:
             raise TypeError("Frequency should be integer (as a row count) or float (as a row percentage)")
 
         # Cython implementor object with input parameters set
-        self._cython_imp = self._get_cython_imp(min_frequency)
+        self.cython_imp = self._get_cython_imp(min_frequency)
 
         # Frequent mining
-        patterns = self._cython_imp.mine()
+        patterns = self.cython_imp.mine()
```

Comment:
wait.. why did this become a public object now?

Reply:
This is also a minor change when I work on the memory clean-up and the encoding implementation with seq2pat backend. This could be changed back to private.